rj_gameplay/stp/formations/diamond_formation.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
+
+import numpy as np
+
+import stp
+from stp.utils.constants import RobotConstants
+
+
+class Diamond_formation(stp.formations.Formations):
```

Comment:
```suggestion
class DiamondFormation(stp.formations.Formations):
```

nit: this is how the rest of our codebase's Python classes are styled, would be nice to match (if you do this don't forget to change the references to this class)